Use yum repo instead of a specific Jenkins version

matsev opened the pull request:
Changed to use official LTS yum repository for Jenkins installation instead of specific Jenkins version.

Ref: https://wiki.jenkins-ci.org/display/JENKINS/Installing+Jenkins+on+Red+Hat+distributions#InstallingJenkinsonRedHatdistributions-Installationofastableversion

michaelwittig commented:
Hi @matsev 
Is it intended that you always install the latest version? I would rather prefer a fixed version otherwise the system can stop working after an instance is terminated ans replace by the auto scaling group because it gets a newer version when it bootstraps. What do you think?

Maybe we could parameterize this behaviour?

matsev commented:
@michaelwittig I see your point, parameterize sounds like a good option. Perhaps with some `Fn::Join` magic so that one does not need to enter the entire path.

matsev commented:
@michaelwittig I have reverted to use rpm for Jenkins installation and added `JenkinsVersion` as an input parameter as suggested, see https://github.com/widdix/aws-cf-templates/compare/master...matsev:patch-1
